lib/momento/get_response.rb

```diff
@@ -1,14 +1,19 @@
 require_relative 'response/error'
 
 module Momento
-  # Responses specific to get.
+  # A response from getting a key.
+  #
+  # Because Momento only stores bytes, the value will be
+  # encoded using Encoding::ASCII_8BIT.
```

Comment:
ASCII as opposed to UTF8? The latter is the standard in other SDKs. Is there a Ruby specific issue here?